msticpy/data/queries/kql_sent_lxsyslog_logon.yaml

```diff
@@ -4,6 +4,34 @@ metadata:
   data_environments: [LogAnalytics]
   data_families: [LinuxSyslog]
   tags: ['linux', 'syslog']
+  aliases:
```

Comment:
We should probably document this feature (couldn't find anything referencing aliases)

Reply:
It's apparently a standard yaml feature. Maybe we should document this whole query template stuff a little better.